### DISCUSSION TITLE
Oculus does not pickup the server?

### DISCUSSION THREAD

**Discussion by Moon-developer** [gerbera/gerbera#1607] [Q&A]
I got Gerbera up and running using a docker image. I know it works because my phone and tv are picking it up but on the Oculus using Skybox/Gallery/Deo does not seem to be working at all.

Not sure if this might help but on windows running media server seems to work just fine. But I'd like to be able to run this on my Ubuntu OS as well :thinking:

**Comment by KarlStraussberger** [comment]
In this case it looks like a network issue.

The core of discovery is SSDP on udp/1900 and the broadcast over 239.255.255.224 (or something like that). See https://docs.gerbera.io/en/latest/run.html#network-setup for details.

Either your occulus is not getting the announce or not sending its search. It's running some windowish thing, so check the firewall.

**Reply by Moon-developer** [reply]
I'm still very new to networking but I ran the following:
```
sudo ufw allow 49152/tcp
sudo ufw allow 1900/tcp
```
Hoping that would help. I think also tried the network setup now that you've linked. But because I don't really know what's happening there I end up getting the following:
```
➜  ~ route add -net 239.0.0.0 netmask 255.0.0.0 eth1
SIOCADDRT: Operation not permitted
➜  ~ sudo route add -net 239.0.0.0 netmask 255.0.0.0 eth1
SIOCADDRT: No such device
```
Answers to fix this from a google search don't make sense to me because of my limited knowledge on this.

**Reply by KarlStraussberger** [reply]
You need to find the name if your network interface (run `ifconfig` or `ip a`). `eht1` does not work in your case.

**Reply by Moon-developer** [reply]
I'm really not having the best of luck setting this up :sweat_smile: 
With your advice I found my network interface and ran the following
```
sudo route add -net 239.0.0.0 netmask 255.0.0.0 wlp7s0
```
which worked then I ran the next command from docs where this one can be found and ran that too.
And just to confirm it's also not a port issue I temporarly disabled my firewall. Unfortuntely still no luck with the Oculus finding the server.

**Reply by KarlStraussberger** [reply]
- Is there some sort of network/firewall setup for your Oculus as well?
- Can you ping the Oculus from the server?
- Being wireless it may suspend network operation - Did you check power management?

**Reply by Moon-developer** [reply]
Just checked on those three.
- No firewall setup on the Oculus from what I can tell
- I was able to ping it from the Linux machine with both being connected to the same network
- Didn't know this might be a thing but I went and changed it to disable it and rebooted. Still the same problem.

Not sure if this helps but I am able to access the server inside the Oculus thru the web browser the same way I do on the Linux machine :thinking: But the DLNA client apps(Skybox/Gallery/DeoVR) don't seem to find it. Such strange behavior. Browser isn't exactly ideal though since it's unfortable to watch movies on.

**Reply by Moon-developer** [reply]
So I setup a DLNA server using miniDLNA and that seems to work. Skybox picks it up. I am stumped as to why this one doesn't show up. I'll keep trying and drop and finding here. Really like the look and feel of Gerbera.
